http/baker-http-server/src/main/scala/com/ing/baker/http/server/javadsl/BakerWithHttpResponse.scala

```diff
@@ -70,6 +70,22 @@ class BakerWithHttpResponse(val baker: Baker, ec: ExecutionContext) extends Lazy
     }
   }
 
+  def getMetaDataFromAddMetaDataRequest(addMetaDataRequestJson: String): Map[String, String] = {
+    parse(addMetaDataRequestJson) match {
+      case Left(_) =>
+        logger.error("Failure parsing bakeRequest")
+        Map.empty[String, String]
+      case Right(json: io.circe.Json) =>
+        json.as[AddMetaDataRequest] match {
+          case Left(_) =>
+            logger.error("Failure parsing bakeRequest")
+            Map.empty[String, String]
```

Comment:
Are we sure that continuing without Metadata might not cause issues while running recipes? Might a more hard failure not be better?

Reply:
Changed it to an exception